src/ristretto/ristretto_keys.rs

```diff
@@ -234,12 +244,6 @@ impl From<u64> for RistrettoSecretKey {
     }
 }
 
-impl From<Scalar> for RistrettoSecretKey {
-    fn from(s: Scalar) -> Self {
-        RistrettoSecretKey(s)
-    }
-}
-
```

Comment:
This is a breaking change. Has it been confirmed that `tari` doesn't use this?

Reply:
I just checked that this doesn't introduce problems against [this PR](https://github.com/tari-project/tari/pull/5636) in `tari` (checking against its main branch isn't possible), so removing the functionality is fine.

Reply:
Probably worth updating the PR description to clarify that this is a breaking API change.